modules/visualizing-external-entities-known-limitations.adoc

```diff
@@ -8,6 +8,5 @@
 [role="_abstract"]
 The following are some known limitations of the Visualizing external entities feature:
 
-* When you enable external IP collection for a cluster, Collector in those clusters report more information to Sensor and to Central. This might create scalability issues if the workload in the cluster communicates with a large number of distinct external peers. It is recommended that you do not enable this feature on clusters with communication patterns involving more than 10,000 distinct external entities.
+* When you enable external IP collection for a cluster, Collector in those clusters report more information to Sensor and to Central. This might create scalability issues if the workload in the cluster communicates with a large number of distinct external peers. It is recommended that you do not enable this feature on clusters with communication patterns involving more than 10,000 distinct external entities. However, if the pattern is mostly ingres or egress, then enabling external IPs only xref:../operating/using-collector-runtime-configuration.adoc#using-collector-runtime-configuration[in the opposite direction] makes it possible to overcome the scaling issue.
```

Comment:
We'll need to remove the xref in the module, they are not allowed.

Reply:
What do you propose we do instead?